primo/utils/solvers.py

```diff
@@ -66,10 +67,10 @@ def get_solver(
     ------
     ValueError
         If an unrecognized solver is provided as an input.
-        Supported solvers include highs, gurobi, scip, and glpk
+        Supported solvers include glpk, gurobi, gurobi_persistent, highs and scip
     """
 
-    if not kwargs:
+    if kwargs:
```

Comment:
Should we remove the `kwargs` argument from the function definition? If we are going to ignore them, why accept them in the first place? Initially I had `kwargs` argument to receive additional solver options. Later, I added `solver_options` argument just for that and forgot to remove `kwargs`.

Reply:
Sounds good to me. I wasn't sure of the original purpose and just thought to update this to keep things safe. I will remove it.